app/src/main/java/com/example/bookwormadventuresdeluxe2/Book.java

```diff
@@ -6,17 +6,22 @@
 import androidx.core.content.res.ResourcesCompat;
 
 import com.example.bookwormadventuresdeluxe2.Utilities.Status;
+import com.google.firebase.firestore.auth.User;
 
 import java.io.Serializable;
+import java.util.ArrayList;
 

```

Comment:
We should probably add a file header comment. Forgot to be on the lookout for that